### PR TITLE
Add interactivity

### PR DESCRIPTION
**Pull request by 2bleO**
-Add 2 new JavaScript files and import them as modules (dragndrop.js & status.js)

-Add event listeners to list items

-Update items object's values for index and completed keys upon user actions

-Store the updated array of items in local storage, so the user gets the correct list values after the page reloads

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- [ ] Dragging a to-do item doesn't switch its position, the position should be changed on dragend.

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/dragndrop.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+export function eventlisteners(arr) {
+  arr.forEach((element) => {
+    const task = document.getElementById(element.index);
+    task.addEventListener('dragstart', (event) => {
+      event.dataTransfer.setData('index', element.index);
+    });
+    task.addEventListener('drop', (event) => {
```
Comment (mikenath223) [inline src/dragndrop.js:7]: - [x] Kindly ensure that dragging a todo item switches its position. Its position should get updated on dragend.

Comment (mikenath223) [reply src/dragndrop.js:7]: - [x] Also ensure that the position switching gets updated on localStorage.

Comment (2bleO) [reply src/dragndrop.js:7]: Sorry@mikenath223 but according to exercise requirements the user gets the correct list values after the page reloads and that's what it do

Comment (2bleO) [reply src/dragndrop.js:7]: > Sorry@mikenath223 but according to exercise requirements the user gets the correct list values after the page reloads and that's what it does

Comment (2bleO) [reply src/dragndrop.js:7]: > * [ ]  Also ensure that the position switching gets updated on localStorage.

Line 19 of dragndrop.js:
window.localStorage.setItem('tasklist', JSON.stringify(arr));

did you try refreshing the page after dragging and dropping?

**Review by carlos-ssh — APPROVED** [review]
Hi @2bleO, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**